### PR TITLE
Update readme externaldns

### PR DESCRIPTION
**Pull request by ideahitme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,16 @@
 [![Coverage Status](https://coveralls.io/repos/github/zalando-incubator/mate/badge.svg?branch=master)](https://coveralls.io/github/zalando-incubator/mate?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/zalando-incubator/mate)](https://goreportcard.com/report/github.com/zalando-incubator/mate)
 
-Mate synchronizes AWS Route53 or Google CloudDNS records with exposed Kubernetes services and ingresses.
+# !!! Update !!!
```
Comment (Raffo) [inline README.md]: I'd go with "Status" or something similar. "Update" assumes that someone is following us which is not a valid assumption. I know it's nitpicking but it's all READMEs are about :P

File: README.md [hunk 2]
```diff
@@ -3,10 +3,16 @@
 [![Coverage Status](https://coveralls.io/repos/github/zalando-incubator/mate/badge.svg?branch=master)](https://coveralls.io/github/zalando-incubator/mate?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/zalando-incubator/mate)](https://goreportcard.com/report/github.com/zalando-incubator/mate)
 
-Mate synchronizes AWS Route53 or Google CloudDNS records with exposed Kubernetes services and ingresses.
+# !!! Update !!! 
+
+Mate will be merged into new External DNS project - https://github.com/kubernetes-incubator/external-dns, which is meant to be compatible with Mate annotations and become a standard way of creating DNS records for K8S.
```
Comment (Raffo) [inline README.md]: K8S -> Kubernetes.

File: README.md (hunk 1 above)
Comment (ideahitme) [reply README.md]: yup :D that's the word i was looking for

**Comment by linki** [comment]
👍

**Pull request merged**